crypto/asn1/a_int.c

```diff
@@ -265,7 +265,7 @@ static int asn1_get_int64(int64_t *pr, const unsigned char *b, size_t blen,
             *pr = -(int64_t)r;
         } else if (r == ABS_INT64_MIN) {
             /* This never happens if INT64_MAX == ABS_INT64_MIN, e.g.
-             * on ones'-complement system. */
+             * on one's-complement system. */
```

Comment:
Please drop this change, it is incorrect. https://en.wikipedia.org/wiki/Ones'_complement

Reply:
> Please drop this change, it is incorrect. https://en.wikipedia.org/wiki/Ones'_complement

@t8m hi, if I undo the change here, then maybe I should change the one in here [one's-complement](https://github.com/openssl/openssl/pull/26095/files#diff-e82cb492031c259cda3912ad288a2d8154b501781daf0374cc5f516077c423baL351). One of them should be changed, and what I want is to keep the comments in consistent when I review the codes.

Reply:
Heh, and Wikipedia isn't consistent: the corresponding page is called [Two's complement](https://en.wikipedia.org/wiki/Two%27s_complement)

Reply:
Okay, since the C standard has

```
— the sign bit has the value −(2^N) (two’s complement);
— the sign bit has the value −(2^N− 1) (ones’ complement).
```

and looking at https://en.wikipedia.org/wiki/Method_of_complements it seems that it should be `NUMBER's complement` when NUMBER is the base (`two`, `ten`) and `NUMBERs' complement` when NUMBER is the largest digit in the base (`one`, `nine`)

Reply:
Yeah, change the other place please. The wrong form is also present in krb5kdf.c and numbers.h.